db/system_keyspace.cc

```diff
@@ -1662,7 +1662,7 @@ template future<> system_keyspace::update_peer_info<utils::UUID>(gms::inet_addre
 template future<> system_keyspace::update_peer_info<net::inet_address>(gms::inet_address ep, sstring column_name, net::inet_address);
 
 template <typename T>
-future<> set_scylla_local_param_as(const sstring& key, const T& value) {
+future<> system_keyspace::set_scylla_local_param_as(const sstring& key, const T& value) {
```

Comment:
nit, s/`methodth`/`method`/ in the title of the commit message.

Reply:
Thanks, fixed